torch/csrc/jit/test_jit.cpp

```diff
@@ -536,8 +536,79 @@ void testADFormulas() {
   }
 }
 
+std::string toString(std::shared_ptr<Graph>& graph) {
+  std::ostringstream s;
+  s << *graph;
+  return s.str();
+}
+
+void testGraphDiff() {
+  auto graph = std::make_shared<Graph>();
+  at::ScalarType s = at::ScalarType::Float;
+  auto type = std::shared_ptr<TensorType>(new TensorType(s, -1, {2, 3, 4}, {12, 4, 1}));
+
+  // Build up a fake graph
+  auto a = SymbolicVariable::asNewInput(*graph, type);
+  auto b = SymbolicVariable::asNewInput(*graph, type);
+  auto c = a * b * a + b;
+  graph->registerOutput(c.value());
+
+  static const char *expected_diff =
+R"(graph(%0 : Float(2, 3, 4)
```

Comment:
As your local testing czar, BOOOOOOOOOO. (Make `--accept` work somehow, don't make me manually fix each of these diffs the next time we have a format change.)